### ISSUE TITLE
Hero animation on Clients page is not accessible

### ISSUE THREAD

**Issue by Tajana-Somogyi** [ALASdoo/alasdoo_com_v2#55]
The animation lasts more than 5 seconds—it should be turned off. 
Color contrast for graphic elements that convey meaning is insufficient.

- [x] replace the video with [the image that has corrected colors](https://drive.google.com/file/d/1hCuRO6m2rCq7azYiiRh8b7SFdpFZm0BR/view?usp=drive_link).

- [x] There needs to be a long description for the map, listing the countries our clients are from:
1. Add link "List of locations of our clients" under the paragraph -> leading to the separate page (List of client's locations]
2. Add alt text to the map: "A world map showing locations of our clients." connected to the link by [the longdesc method](https://www.w3.org/WAI/tutorials/images/complex/#:~:text=figcaption%3E%0A%3C/figure%3E-,Approach%204%3A%20Providing%20a%20link%20to%20the%20long%20description%20via%20longdesc,-Accessibility%20Support%3A).
3. Make a separate page that only shows [the list of client's locations](https://docs.google.com/document/d/16h9m8AYLsCDR-i0nOslbAVhq2ufRy0Dd6JryjS_HDyM/edit?usp=sharing): "Alas clients' locations" (include it in the site map)

- [x] test with screen reader

- example: https://dequeuniversity.com/tips/long-descriptions/

**Comment by Tajana-Somogyi** [comment]
Conclusion: all browsers do not support the longdesc method, thats why we are changing the alt text to "A world map showing locations of our clients. See all locations in the following link to the list."
